src/sgame/sg_admin.cpp

```diff
@@ -863,11 +863,15 @@ static bool admin_higher( gentity_t *admin, gentity_t *victim )
 	                           victim->client->pers.admin );
 }
 
-static void admin_writeconfig_string( char *s, fileHandle_t f )
+static void admin_writeconfig_string( Str::StringRef s, fileHandle_t f )
 {
-	if ( s[ 0 ] )
+	if ( !s.empty() )
 	{
-		trap_FS_Write( s, strlen( s ), f );
+		if ( ( s.front() & 0xFF ) <= ' ' || ( s.back() & 0xFF ) <= ' ' )
+		{
+			Log::Warn( "String '%s' cannot be serialized in admin config", s );
```

Comment:
return, or update warning to note that the output maybe mangled.

Reply:
Done